backend/.pre-commit-config.yaml

```diff
@@ -45,3 +45,7 @@ repos:
           - --disable=R0902
           - --disable=R0903
           - --disable=C0103
+          - --disable=R0913
+          - --disable=C0327
+          - --disable=C0305
```

Comment:
Why were some pre-commit hooks disabled ?

Reply:
I disabled the `RO913: Too many arguments (6/5)` hook because it was complaining about the update( ) function in db.py (of which the 6 arguments it has are all very much needed)

but as for the rest, I'll enable them - my bad